src/test/java/seedu/sellsavvy/logic/commands/ordercommands/DeleteOrderCommandTest.java

```diff
@@ -0,0 +1,96 @@
+import static seedu.sellsavvy.testutil.TypicalIndexes.INDEX_FIRST_ORDER;
+import static seedu.sellsavvy.testutil.TypicalIndexes.INDEX_SECOND_ORDER;
+import static seedu.sellsavvy.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.sellsavvy.commons.core.index.Index;
+import seedu.sellsavvy.logic.Messages;
+import seedu.sellsavvy.model.Model;
+import seedu.sellsavvy.model.ModelManager;
+import seedu.sellsavvy.model.UserPrefs;
+import seedu.sellsavvy.model.order.Order;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code DeleteOrderCommand}.
+ */
+public class DeleteOrderCommandTest {
+
+    private Model model;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    }
+
+    @Test
+    public void execute_validIndexOrderList_success() {
+        Order order = model.getSelectedPerson().get().getOrderList().get(INDEX_FIRST_ORDER.getZeroBased());
+        DeleteOrderCommand deleteOrderCommand = new DeleteOrderCommand(INDEX_FIRST_ORDER);
+
+        String expectedMessage = String.format(DeleteOrderCommand.MESSAGE_DELETE_ORDER_SUCCESS,
+                order.getItem());
+
+        ModelManager expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        expectedModel.getSelectedPerson().get().getOrderList().remove(INDEX_FIRST_ORDER);
+
+        assertCommandSuccess(deleteOrderCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_invalidIndexOrderList_throwsCommandException() {
+        Index outOfBoundIndex = Index.fromOneBased(
+                model.getSelectedPerson().get().getOrderList().size() + 1);
+        DeleteOrderCommand deleteOrderCommand = new DeleteOrderCommand(outOfBoundIndex);
+
+        assertCommandFailure(deleteOrderCommand, model, Messages.MESSAGE_INVALID_ORDER_DISPLAYED_INDEX);
+    }
+
+    @Test
+    public void equals() {
+        DeleteOrderCommand deleteFirstCommand = new DeleteOrderCommand(INDEX_FIRST_ORDER);
+        DeleteOrderCommand deleteSecondCommand = new DeleteOrderCommand(INDEX_SECOND_ORDER);
+
+        // same object -> returns true
+        assertTrue(deleteFirstCommand.equals(deleteFirstCommand));
+
+        // same values -> returns true
+        DeleteOrderCommand deleteFirstCommandCopy = new DeleteOrderCommand(INDEX_FIRST_ORDER);
+        assertTrue(deleteFirstCommand.equals(deleteFirstCommandCopy));
+
+        // different types -> returns false
+        assertFalse(deleteFirstCommand.equals(1));
+
+        // null -> returns false
+        assertFalse(deleteFirstCommand.equals(null));
+
+        // different person -> returns false
+        assertFalse(deleteFirstCommand.equals(deleteSecondCommand));
+    }
+
+    @Test
+    public void toStringMethod() {
+        Index targetIndex = Index.fromOneBased(1);
```

Comment:
isn't this just the first index u created in typical index which u already imported?

Reply:
```suggestion
        Index targetIndex = INDEX_FIRST_ORDER;
```
Maya be different depending on the change made for https://github.com/AY2425S1-CS2103T-F14a-2/tp/pull/82/files#r1804048146